### PR TITLE
build: Allow building gdb with and without python

### PR DESCRIPTION
**Pull request by guyush1**

### CODE REVIEW

**Comment by guyush1** [comment]
This pr closes #25

**Comment by guyush1** [comment]
For some reason ci-cd is not running. @roddyrap  any ideas  why :) ?

**Comment by roddyrap** [comment]
Good question, I will check

**Comment by guyush1** [comment]
I need to add the with-python build targets to the ci-cd (both release and regular pipelines)
edit: done!

**Comment by guyush1** [comment]
> Good question, I will check

@roddyrap 
No need, i just remembered i needed to update the ci-cd anyways since we need to include both build and build-with-python options now. After i pushed the change, ci-cd was triggered. It was probably a bug the first time...

**Review by roddyrap — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -62,4 +81,4 @@ clean: clean-git-packages
 	rm -rf build
 # Kill and remove all containers of image gdb-static
 	docker ps -a | grep -P "^[a-f0-9]+\s+gdb-static\s+" | awk '{print $$1}' | xargs docker rm -f 2>/dev/null || true
-	docker rmi -f gdb-static 2>/dev/null || true
+# docker rmi -f gdb-static 2>/dev/null || true
```
Comment (guyush1) [inline Makefile]: No, do not comment this out

Comment (roddyrap) [reply Makefile]: lol

Comment (guyush1) [reply Makefile]: fixed

**Pull request merged**